frontend/src/modules/dataSources/components/DataSourceManager/TestConnection.jsx

```diff
@@ -35,6 +36,8 @@ export const TestConnection = ({
 
   function testDataSource() {
     setTestingStatus(true);
+    posthogHelper.captureEvent('test_connection_datasource', { dataSource: kind, pluginId });
```

Comment:
Shall we match the properties (source) ?

`    posthog.capture('test_connection_datasource', { dataSource: kind, appId });
`

Copied from the current cloud code